main.tf

```diff
@@ -42,6 +45,13 @@ locals {
       "Name" = var.append_schedule_postfix ? "${replace(index, "_", "-")}-schedule" : index
     })
   ])
+  eventbridge_pipes = flatten([
+    for index, pipe in var.pipes :
+    merge(pipe, {
+      "name" = index
+      "Name" = var.append_pipe_postfix ? "${replace(index, "_", "-")}-pipe" : index
+    })
```

Comment:
does this need to be guarded with a `if local.create_pipes`?

Reply:
It is guarded when it is used in the `for_each`